extensions/ql-vscode/src/remote-queries/export-results.ts

```diff
@@ -110,8 +110,8 @@ export async function exportResultsToGist(
 const buildGistDescription = (query: RemoteQuery, analysesResults: AnalysisResults[]) => {
   const resultCount = sumAnalysesResults(analysesResults);
   const resultLabel = pluralize(resultCount, 'result', 'results');
```

Comment:
Yes, it can. But the [pluralize](https://github.com/github/vscode-codeql/commit/9c598c2f0665a15429cdc5d2578ce477df869f83) method handles that:
```
export function pluralize(numItems: number | undefined, singular: string, plural: string): string {
  return numItems ? `${numItems} ${numItems === 1 ? singular : plural}` : '';
}
```

For `repositoryCount`, we also needed to know when to skip the parentheses, but that's not a requirement for `resultCount`

Reply:
Ah gotcha! Thanks.